crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -4385,6 +4345,72 @@
           }
         }
       },
+      "NetworkMetrics": {
```

Comment:
I'd personally split this into two APIs: one for current data, one for historical (count) data.

Reply:
I am trying not to introduce too many new api just for metrics, also theses metrics are small and the queries are cached, so it's no harm returning all of them in one call.